doc/getting-started/building-the-node-using-nix.md

```diff
@@ -4,6 +4,7 @@ The [Nix Package Manager][nix] can be installed on most Linux distributions by d
 running the installation script:
 ```
 curl -L https://nixos.org/nix/install > install-nix.sh
+chmod +x install-nix.sh
```

Comment:
Why not one-liner: `curl -L https://nixos.org/nix/install | sh`

Reply:
This would work as well. But I am not here to propose the best solution. I am only want to show that the current documentation has a bug. You can not execute the shell script if it doesn't have the executable bit set. If you want you can propose a merge request with your solution and then I take mine back.